container/overlay/tomcat/webapps/mp/WEB-INF/web.xml

```diff
@@ -36,7 +36,7 @@
     <servlet-class>org.sdo.demo.RelayServlet</servlet-class>
     <init-param>
       <param-name>forward-to</param-name>
-      <param-value>rps</param-value>
+      <param-value>8001</param-value>
```

Comment:
Comment:
Use value "rps" for using PRI Rendezvous service. Use value "8001" to use On-Prem Rendezvous-Service.